packages/babel-generator/src/buffer.js

```diff
@@ -38,10 +38,25 @@ export default class Buffer {
   get(): Object {
     this._flush();
 
-    return {
+    const map = this._map;
+    const result = {
       code: trimEnd(this._buf.join("")),
-      map: this._map ? this._map.get() : null,
+      map: null,
+      rawMappings: map && map.getRawMappings(),
     };
+
+    if (map) {
+      Object.defineProperty(result, "map", {
+        get() {
+          return this.map = map.get();
```

Comment:
I wonder why the tests weren’t failing for this. If I set `configurable` and `enumerable` to `false` explicitly, it will throw, although that should be equivalent to elision.